### DISCUSSION TITLE
Why does HyperHDR use my webcam?

### DISCUSSION THREAD

**Discussion by JoshAtticus** [awawa-dev/HyperHDR#460] [General]
Current;y covering my camera in masking tape right now, only installed HyperHDR and boom, the camera led turned on. ran `sudo lsof /dev/video0` and confirmed it is using it 

josh@Josh-Alt-Laptop:~$ sudo lsof /dev/video0
[sudo] password for josh: 
lsof: WARNING: can't stat() fuse.gvfsd-fuse file system /run/user/1000/gvfs
      Output information may be incomplete.
lsof: WARNING: can't stat() fuse.portal file system /run/user/1000/doc
      Output information may be incomplete.
COMMAND   PID USER   FD   TYPE DEVICE SIZE/OFF NODE NAME
hyperhdr 3951 josh  mem    CHR   81,0           569 /dev/video0
hyperhdr 3951 josh   17u   CHR   81,0      0t0  569 /dev/video0

Please tell me why this is happening.

**Comment by JoshAtticus** [comment]
`sudo killall hyperhdr` makes the camera LED turn off.

**Comment by awawa-dev** [comment]
Just disable grabber (camera is auto-detected as a video device at /dev/video0, same as grabbers) in the video control: same page where you have the grabber configuration. Or manually select your video grabber from the list if you have one.

![obraz](https://user-images.githubusercontent.com/69086569/207181055-4fb0701d-6518-47e4-afa2-047be100b6d9.png)

**Comment by awawa-dev** [comment]
not an issue, moving to the discussion
